File: packages/sdk-java/src/main/java/com/pillar/resources/ReportTemplates.java
package com.pillar.resources;
public final class ReportTemplates { private final Resource resource; public ReportTemplates(Resource resource){ this.resource=resource; } public Object create(Object body,String idempotencyKey){ return resource; } public Object list(){ return resource; } public Object run(String id,String idempotencyKey){ return resource; } }

File: packages/sdk-python/pillar/resources/report_templates.py
class ReportTemplates:
    def __init__(self, client): self.client=client
    def create(self, data, idempotency_key): return self.client.request('POST','/v1/report_templates', json=data, idempotency_key=idempotency_key)
    def list(self, **params): return self.client.request('GET','/v1/report_templates', params=params)
    def get(self, id): return self.client.request('GET',f'/v1/report_templates/{id}')
    def run(self, id, idempotency_key): return self.client.request('POST',f'/v1/report_templates/{id}/run', idempotency_key=idempotency_key)
